modules/simple_forms_api/app/models/simple_forms_api/vha_10_10d.rb

```diff
@@ -24,10 +24,15 @@ def metadata
 
     def handle_attachments(file_path)
       attachments = get_attachments
+      new_file_names = [file_path]
```

Comment:
@cloudmagic80 I noticed in `uploads_controller` that we have this in place `'vba_40_0247', 'vha_10_10d', 'vba_40_10007'` for the case to hit this method. Will this update break the `vba` forms or do we want this functionality for forms outside of the ones we are working on.